pagopa/io-app
# [#172124636] show debug dev mode only

**Short description:**
The Developer section in the Profile Screen is now visible only if the application is executed in development environment (`isDevEnv==true`).

**List of changes proposed in this pull request:**
Grouping the Developer Section and show only if ìsDevEnv==true`.

**How to test:**
Change the value to `IsDevEnv` to see the different behaviour in the rendering of the Profile Screen.
